allennlp/predictors/predictor.py

```diff
@@ -57,17 +63,157 @@ def predict_json(self, inputs: JsonDict) -> JsonDict:
         instance = self._json_to_instance(inputs)
         return self.predict_instance(instance)
 
+    def interpret_from_json(self, inputs: JsonDict) -> JsonDict:
+        """
+        Uses the gradients from :func:`get_gradients` to provide 
+        normalized interpretations for specific models. 
+
+        Raises
+        ------
+        NotImplementedError
+            This method should be overriden by the specific predictor class of each
+            predictor.
+        """
+        raise NotImplementedError
```

Comment:
This needs to raise a `RuntimeError`, not a `NotImplementedError`.  Same for the other methods you added.  This is because we don't want to force all predictors to implement this - we're getting pylint errors about abstract methods not being overridden.  The `RuntimeError` should say something like "you need to implement this method if you want to give model interpretations".